airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -335,7 +335,8 @@ def _send_request(self, request: requests.PreparedRequest, request_kwargs: Mappi
         backoff_handler = default_backoff_handler(max_tries=max_tries, factor=self.retry_factor)
         return backoff_handler(user_backoff_handler)(request, request_kwargs)
 
-    def parse_response_error_message(self, response: requests.Response) -> Optional[str]:
+    @classmethod
```

Comment:
turn this into a classmethod so it can be shared with https://github.com/airbytehq/airbyte/pull/14330/files#diff-64c2772cfbda6debaa3a8d4a26598f9a9a194d4ef17cdc61537675a9e0db43baR31-R35